.travis.yml

```diff
@@ -1,12 +1,26 @@
 language: go
 
 go:
-    - 1.4
-    - 1.5
-    - 1.6
-    - 1.7
-    - 1.8
-    - 1.9
-    - tip
+  - "1.4.x"
+  - "1.5.x"
+  - "1.6.x"
+  - "1.7.x"
+  - "1.8.x"
+  - "1.9.x"
+  - "1.10.x"
+  - "1.11.x"
+  - "1.12.x"
+  - "1.13.x"
+  - tip
+
+os:
+  - linux
+  - osx
```

Comment:
Can we please restrict this PR to this content only. The rest is unrelated to what the PR proposes and I'd rather get the main intention in right now than keep discussing unrelated bits here.

Reply:
Sounds good, I will refine the travis build changes.

Reply:
@niemeyer Updated, now the syntax should be using the latest versions of each golang lines.

Reply:
@chenrui333 I think what was meant by that was to not include the go.mod/go.sum changes in this PR

Reply:
Interesting, so we just need travis build config changes?

Reply:
Presumably, since the pr into the v3 branch I made doesn't touch the go.mod/go.sum files, and they're the same between the branches